config/initializers/swagger_blocks.rb

```diff
@@ -0,0 +1,34 @@
+module ApiFlashcards
+  include Swagger::Blocks
+    swagger_root do
+      key :swagger, '2.0'
+      info do
+        key :version, '1.0.0'
+        key :title, 'Flashcards API'
+        key :description, 'A sample API for Flashcards App'
+      end
+      tag do
+        key :name, 'Cards'
+      end
+      tag do
+        key :name, 'Review'
+      end
+      key :host, 'localhost:3000'
+      key :basePath, '/api/v1'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.